CI/flatpak/com.obsproject.Studio.json

```diff
@@ -263,6 +263,32 @@
         }
       ]
     },
+    {
+      "name": "pipewire",
```

Comment:
Ah, I just noticed you added this between `cef` and `obs` - could you please move this to after `jack2`? That way we keep dependencies closer to their dependents. (There's actually a better way to declare dependencies between sources, but I'll do that separately in a follow-up pull request.)